client/src/components/landing/landing.css

```diff
@@ -406,12 +405,17 @@ figcaption.caption {
   .landing-top-two-column {
     flex-direction: row;
     align-items: center;
+    width: min(100%, 1300px);
```

Comment:
I know this is from the footer CSS, but could you help check if we can replace these width values with the Col component? We don't need pixel-perfect, similar width values would be good enough.

Doc: https://opensource.freecodecamp.org/ui/?path=/docs/components-col--docs

`LandingTop` already uses `Col` but doesn’t pass any column props, and the `Footer` component doesn't use `Container`/`Row`/`Col` at all. Using the same layout components across sections would give us better consistency.

I realize this is out of scope and this part of the codebase isn't in great shape. Please treat this as a quick check to see if it's a straightforward change. If it looks more involved, we can have a proper refactor later :)